test/Interop/Cxx/extern-var/extern-var-irgen.swift

```diff
@@ -0,0 +1,19 @@
+// Test that global variables are handled properly by the ClangImporter.
+
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -I %S/Inputs -emit-ir -o %t/extern-var.ir -Xfrontend -enable-cxx-interop
+// RUN: %FileCheck < %t/extern-var.ir %s
+
+// CHECK: @counter = external global i32, align 4
+//FIXME mangle non-top-level var names to prevent name collisions
+// and check @_ZN10Namespaced7counterE = external global i32, align 4
+
+import ExternVar
+
+func getCounter() -> CInt {
+  return counter
+}
+
+func getNamespacedCounter() -> CInt {
+  return Namespaced.counter
+}
```

Comment:
Could you also add CHECK lines that verify the IR for reading and writing the variables?

Reply:
Done.